README.md

```diff
@@ -0,0 +1,13 @@
+# How to update
+
+  - The repo of this online tutorial content is at https://github.com/mozilla-b2g/casting-tutorial.
+    Please send pull request to this specific repo.
+
+  - Accroding to the bug 1215006: https://bugzilla.mozilla.org/show_bug.cgi?id=1215006,
+    we serve the turotial content at https://casttv.services.mozilla.com/.
+    Please file a bug to jthomas@mozilla.com to update the website.
+
+# The Fling tutorial
```

Comment:
The app folder and repo name is fling-tutorial. The displayed app name is "Send to TV". Since the displayed app name could change and to align with the app folder and repo name, we use "Fling tutorial" here.